packages/labextension/package.json

```diff
@@ -42,24 +41,25 @@
     "watch": "tsc --build --watch"
   },
   "dependencies": {
-    "@jupyterlab/apputils": "^2 || ^3",
-    "@jupyterlab/coreutils": "^4 || ^5",
-    "@jupyterlab/nbformat": "^2 || ^3",
-    "@jupyterlab/notebook": "^2 || ^3",
-    "@jupyterlab/rendermime": "^2 || ^3",
-    "@jupyterlab/services": "^5 || ^6",
-    "@jupyterlab/settingregistry": "^2 || ^3",
-    "@lumino/algorithm": "^1.1.2",
-    "@lumino/coreutils": "^1.3.0",
-    "@lumino/disposable": "^1.1.2",
-    "@lumino/widgets": "^1.6.0",
+    "@jupyterlab/application": "^4.0.0",
```

Comment:
I see you moved some packages from "devDependencies" to "dependencies" here. Was this intentional?

Reply:
Packages will be moved as it was previously.